### PR TITLE
[PS-2350] update searchbyid api path 

### PR DESCRIPTION
**Pull request by swVisenze**

### CODE REVIEW

File: doc/ProductSearch.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ With the release of ViSenze's Catalog system, ViSearch Android SDK will now incl
 2. [Initialization](#2-initialization)
 3. [Solution APIs](#3-solution-apis)
      - 3.1 [Search by Image](#31-search-by-image)
-     - 3.2 [Search by ID](#32-search-by-id)
+     - 3.2 [Search by ID/recommendation](#32-search-by-id)
```
Comment (thehung111) [inline doc/ProductSearch.md]: change this to Recommendations

File: doc/ProductSearch.md [hunk 2]
```diff
@@ -173,9 +173,9 @@ public void OnImageCaptured(Image image, String imagePath) {
 
 > Image File refers to an actual file with bytes representing the image (i.e. opened from file upload, or taken from camera). The example above is in a scenario where the android camera captures an image.
 
-### 3.2 Search By ID
+### 3.2 Search By ID/recommendations
```
Comment (thehung111) [inline doc/ProductSearch.md]: change to Recommendations

File: doc/ProductSearch.md (hunk 1 above)
Comment (thehung111) [reply doc/ProductSearch.md]: update the path to `#32-recommendations`

File: doc/ProductSearch.md [hunk 3]
```diff
@@ -173,9 +173,9 @@ public void OnImageCaptured(Image image, String imagePath) {
...
 
-GET /product/search_by_id/{product_id}
+GET /product/recommendations/{product_id}
 
 This Search By ID is NOT the same ID mentioned in [Search By Image](#31-search-by-image)'s ID. This ID refers to the product's ID and not the image's ID.
```
Comment (thehung111) [inline doc/ProductSearch.md:180]: update this text

File: doc/ProductSearch.md [hunk 4]
```diff
@@ -192,6 +192,15 @@ public void yourFunction(ProductResponse yourPriorResponse)
             // LOGIC HERE
         }
     });
+    /* or use recommendations 
+    ps.recommendations(params, new ProductSearch.ResultListener()) {
```
Comment (thehung111) [inline doc/ProductSearch.md:189]: remove the searchbyid path and replace with recommendations

File: doc/ProductSearch.md [hunk 5]
```diff
@@ -208,7 +217,7 @@ The example above assumes that you have stored a prior successful ProductRespons
 Depending on which API function you call, it requires their own parameters both of which is extends off the [BaseProductSearchParams](#41-baseproductsearchparams) class:
 
 - [Search by Image](#31-search-by-image) uses [ProductSearchByImageParams](#42-productsearchbyimageparams)
-- [Search by ID](#32-search-by-id) uses [ProductSearchByIdParams](#43-productsearchbyidparams)
+- [Search by ID/recommendations](#32-search-by-id) uses [ProductSearchByIdParams](#43-productsearchbyidparams)
```
Comment (thehung111) [inline doc/ProductSearch.md]: same comment as above

File: doc/ProductSearch.md [hunk 6]
```diff
@@ -324,7 +333,7 @@ public void onSearchResult(ProductResponse response, ErrorData error) {
 
 | Name | Type | Description |
 |:---|:---|:---|
-| productId | String | The product's ID which can be used in [Search by ID](#32-search-by-id). |
+| productId | String | The product's ID which can be used in [Search by ID/recommendations](#32-search-by-id). |
```
Comment (thehung111) [inline doc/ProductSearch.md]: same comment

File: doc/ProductSearch.md (hunk 1 above)
Comment (swVisenze) [reply doc/ProductSearch.md]: changed

File: doc/ProductSearch.md (hunk 2 above)
Comment (swVisenze) [reply doc/ProductSearch.md]: changed

File: doc/ProductSearch.md (hunk 3 above)
Comment (swVisenze) [reply doc/ProductSearch.md:180]: changed

File: doc/ProductSearch.md (hunk 4 above)
Comment (swVisenze) [reply doc/ProductSearch.md:189]: changed

File: doc/ProductSearch.md (hunk 5 above)
Comment (swVisenze) [reply doc/ProductSearch.md]: changed

File: doc/ProductSearch.md (hunk 6 above)
Comment (swVisenze) [reply doc/ProductSearch.md]: changed

**Review by thehung111 — APPROVED** [review]

**Pull request merged**